server/api/assessment/export.ts

```diff
@@ -1,7 +1,7 @@
 import { Express, Response, Request } from 'express'
 import JSZip from 'jszip'
 import { ApiAuthMiddleware } from '@server/api/middleware'
-import * as ExportService from '@server/service/assessment/exportService'
+import * as ExportService from '@server/service/assessment/_legacy/exportService'
```

Comment:
why renaming the service? review might take longer

Reply:
Just moved under _legacy, no renaming!